tests/unit/tasks/test_base_task.py

```diff
@@ -112,9 +109,11 @@ def test_children_property_no_structure(self, task):
         assert len(parent.children) == 3
 
     def test_execute_publish_events(self, task):
-        task.execute()
+        handler = Mock()
+        with EventListener(handler):
+            task.execute()
 
-        assert EventBus.event_listeners[0].handler.call_count == 2
+            assert handler.call_count == 2
```

Comment:
This line doesn't need to be indented, right? (Like it should be the same inside or outside the context manager scope, right?)